sonar-db/src/main/resources/org/sonar/db/ce/CeActivityMapper.xml

```diff
@@ -18,12 +18,26 @@
     ca.updated_at as updatedAt,
     ca.is_last as isLast,
     ca.is_last_key as isLastKey,
-    ca.execution_time_ms as executionTimeMs
+    ca.execution_time_ms as executionTimeMs,
+    ca.error_message as errorMessage,
+    <choose>
```

Comment:
But I would then retrieve the content of the CLOB, which I don't want for obvious performance reasons
testing whether CLOB is null or not does not require to retrieve the CLOB